framework/tests/value/ValueCast.java

```diff
@@ -0,0 +1,11 @@
+// Test case for issue 1264: https://github.com/typetools/checker-framework/issues/1264
```

Comment:
This is an implicit cast from `short` to `int`. I would expect that a solution to #1264 would also address this issue. If you really think it's a different issue, we can add it as one, but I think that they are two sides of the same coin.